proto/build.rs

```diff
@@ -22,7 +22,7 @@ fn main() -> miette::Result<()> {
     tonic_build::configure()
         .file_descriptor_set_path(&file_descriptor_path)
         .skip_protoc_run()
-        .out_dir("src")
+        .out_dir("src/generated")
```

Comment:
Moved the files to a folder because of the `rustfmt` configuration. It should be disabled for the generated code, but not for the rest.